README.md

```diff
@@ -176,7 +187,7 @@ Note: Upstart/SysV init based OS types are not supported.
 
 ## Container Runtime Notes
 
-- The list of available docker version is 18.09, 19.03 and 20.10. The recommended docker version is 20.10. The kubelet might break on docker's non-standard version numbering (it no longer uses semantic versioning). To ensure auto-updates don't break your cluster look into e.g. yum versionlock plugin or apt pin).
+- Available Docker versions are 18.09, 19.03 and 20.10. The recommended Docker version is 20.10. `Kubelet` might break on docker's non-standard version numbering (it no longer uses semantic versioning). To ensure auto-updates don't break your cluster look into e.g. the YUM  ``versionlock` plugin or ``apt pin``).
```

Comment:
There is an invalid markdown format like:
```
$ markdownlint $(find . -name '*.md' | grep -vF './.git') --ignore docs/_sidebar.md --ignore contrib/dind/README.md
./README.md:190:276 MD038/no-space-in-code Spaces inside code span elements [Context: "``versionlock` plugin or ``"]
```

Reply:
TIL about `markdownlint`.  The above was a result of a missing backquote.  I pushed a fix along with fixes for a couple of unrelated lint complaints.